tests/devices/test_default_qutrit_mixed.py

```diff
@@ -1270,3 +1270,305 @@ def qfunc(x, y):
         grad_jit = jax.grad(qnode_jit)(x, y)
 
         assert qml.math.allclose(grad, grad_jit)
+
+
+@pytest.mark.parametrize("nr_wires", [2, 3])
+class TestReadoutError:
```

Comment:
suggestion: standardize class naming

This class name doesn't match the usage of measurement error earlier. However, you did ask whether we should call this measurement or readout error; seeing this class, I think I prefer "readout error".

Reply:
Ok, I will change measurement_error to readout_error across the PR.